app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, counter_cache: :comment_counter
+
+  def increment_comment_count
+    post.increment!(:comment_counter)
+  end
```

Comment:
- [ ] Optional: Just as I advised above, `after_save :update_comments_counter`

Reply:
- [ ] Optional

As suggested by the previous reviewer, consider using rails active callback `after_save` for the method `increment_comment_count`. Doing this will save you a lot of headaches later since you won't have to call this method every time you create a comment.